charts/spire/charts/spire-server/README.md

```diff
@@ -28,6 +28,56 @@ A Helm chart to install the SPIRE server.
 
 * <https://github.com/spiffe/helm-charts/tree/main/charts/spire>
 
+## Tornjak
+
+Tornjak is the UI and Control Plane for SPIRE [https://github.com/spiffe/tornjak](https://github.com/spiffe/tornjak)
```

Comment:
Should we mention the UI part in this README? 

Technically it is part of another chart. Probably better to document this in the spire chart or the tornjak-ui chart. And refer int this documentation to that chart.